public/css/contactEmail.css

```diff
@@ -0,0 +1,324 @@
+
```

Comment:
Is this file a new file?

Reply:
Yes. That email that josh did. I checked the email2 and it's just a congratulatory message.